packages/react/src/base-chart.js

```diff
@@ -7,6 +7,20 @@ export default class BaseChart extends React.Component {
 		this.data = props.data;
 		this.options = props.options;
 
+		// Width prop is mandatory for the wrappers
```

Comment:
Why are width and height "mandatory for the wrappers"?

Reply:
This is an issue with frameworks such as React & Angular. The chart holder does not start at its ideal size initially, and re-renders multiple times to get to the final chart size, an as a result of that, `resizeChart()` is called multiple times, making the initial render seem animated.
![image](https://user-images.githubusercontent.com/14989804/49159952-a8564180-f2f3-11e8-80ba-d2bc49ccc12b.png)

We can use `react-sizeme` solves this by initially rendering the component invisibly, then re-rendering to the full size, or another approach that **most** charting frameworks use in React is to make width & height mandatory.